components/Benefit/Benefit.js

```diff
@@ -0,0 +1,105 @@
+import styled from "styled-components";
+import freeshippingicon from "./img/freeshippingicon.svg";
```

Comment:
Move all svgs to the icons.js in assets. Also separate library and file imports.